common/Fabric-Install-Utilities.psm1

```diff
@@ -63,7 +62,7 @@ function New-AppPool($appName, $userName, $credential){
 		$appPool = Get-Item $appName
 	}
 
-	if(![string]::IsNullOrEmpty($userName) -and $credential -ne $null)
+	if(!([string]::IsNullOrEmpty($userName) -and $credential -ne $null))
```

Comment:
I think these added parens change the logic of this if statement...we're negating the result of the whole statment instead of just the first portion...

Reply:
You are right. I fixed this and tested an install again.